test/utils/image/manifest.go

```diff
@@ -0,0 +1,141 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package image
+
+import (
+	"fmt"
+	"runtime"
+)
+
+const (
+	e2eImagesRegistry  = "gcr.io/kubernetes-e2e-test-images"
+	BusyBox            = "busybox"
```

Comment:
I think we should use Busybox from Docker Hub, yes. That will get us security patches automatically. I think Docker Hub is more reliable now... or?
cc @fejta 

@ixdy We'll ideally, we'd use manifest lists. Plan B might be to lookup the API server platform by hitting `/version` on every image get, but that's kinda painful. Plan C (which this is, and is what I think we should stick with now) is to use `runtime.GOARCH` from the e2e binary. If you have an arm cluster, you have to make sure you call e2e tests from an arm machine as well, etc.